### DISCUSSION TITLE
Issue after creating a list from modal/pop-up screen

### DISCUSSION THREAD

**Discussion by guibranco** [community/community#58574] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If you starred a repository and try to add it to a new list, creating new one, after you create the list, the expected behavior is that the interface creates the list, add the repository to it, and closes the modal. 

Actually it freezes on "creating" step, (in background the list is created and the repository assigned to it correctly).

![image](https://github.com/community/community/assets/3362854/3cd5c373-958b-4f2d-8ac5-240ce7c709e7)

In the `browser devTools` there are no errors, just a warning:
![image](https://github.com/community/community/assets/3362854/37f796b7-6e54-4e79-97db-a22d2c265899)
![image](https://github.com/community/community/assets/3362854/75a5f1cc-6837-4547-8720-1e62de66adc4)

This also happens if you try to create the list from the starred repositories lists on your profile (looks like is `create list` component bug instead a random bug caused by some other component in a specific page)

---

Steps to reproduce: 

- Go to any repository page (or to your starred repositories page).
- Click on the button to add it to starred list or to a specific list on the dropdown 
- Type a name of a list that doesn't exists
-  Click on the `+ create list "new list name"`
- Adjust the list name (in my cases, I added an emoticon to the end of the name)
- Type any valid description for the list
- Click on the green button `Create`
- The modal never closes or end the processing step

**Comment by DSDmark** [comment]


**Reply by guibranco** [reply]
As I mentioned, there is no error reported in DevTools.
If my connection dropped while executing the action, it should generate an error in the console, and I would be able to see the error on the network tab/panel as well, this is not the case.

More over, as I told, that was not the first time it occur, even in different pages (not only direct in the repository's page, but also in the starred list's page), even using different devices. 
I had the same issue using a Mac from my previously company, I had the issue using my personal laptop (Windows), and I had the issue with my current company's laptop (that one from the screenshots above) that is also a Windows.

**Reply by Andreea34** [reply]
I came across this issue too. The "Create list" pop-up modal persists even if you navigate to different URLs (see screenshot below of what gets displayed after clicking the "Share feedback and report bugs" link in the modal). I highly doubt this is symptomatic of a network issue.

![image](https://github.com/community/community/assets/26445466/ad624220-2128-4538-ad47-b3bbe6c6a044)

**Comment by krtffl** [comment]
i've just experienced the same problem twice. both times i tried to create the list from the modal. it just gets frozen, no response.

![image](https://github.com/community/community/assets/96619540/8172e4fa-efef-4bf3-b722-a62be40ee94d)

close button is responsive. on close, the list cannot be seen yet. however, after reloading the new list appears as expected and seems to be properly created

![image](https://github.com/community/community/assets/96619540/09c0d496-8f92-4260-a27c-ab4f921ae99b)

as mentioned, the repo is properly added to the list too on its creation.

**Reply by Andreea34** [reply]
Right now, the only thing you can do is close/dismiss the modal pop-up manually using the "X" button, and refresh the page in your browser to see the updated list information.

**Reply by krtffl** [reply]
you can just go to the top of the page on your stars and you'll see all the lists, you can edit the names there too

**Reply by Rigavinil201** [reply]
Add

**Reply by Rigavinil201** [reply]
None

**Comment by juanmanso** [comment]
Same issue here too!

<details><summary>Click to show screenshot</summary> 
<img width="1307" alt="image" src="https://github.com/community/community/assets/21087992/2d5df1ae-6193-4bbb-9500-8edd9d0b0e96">
</details>

**Comment by rpnunez** [comment]
This has also happened to me. Firefox on Fedora. Has also happened in Windows.

![Screenshot at 2023-06-24 16-01-04](https://github.com/community/community/assets/2415570/eedfc5c5-8861-4e03-baea-7b89cbc0f958)

**Comment by Rigavinil201** [comment]
Add

**Comment by ChienNQuang** [comment]
I have the same bug every time creating a new list.

![image](https://github.com/community/community/assets/87883163/ed0ddcb6-a911-4352-9d14-a3f2ab6284ab)

**Comment by JokerQyou** [comment]
Same issue.

![image](https://github.com/community/community/assets/1465267/8822acc5-3d1b-43a2-93d8-154c389c7588)

What I noticed is: there's a POST request being made to `https://github.com/stars/USERNAME/lists`, and the server seems to respond with a full HTML page (instead of a short HTML fragment or some JSON data), might be related to this bug.

It's over three weeks since this bug occurred. Why is it still not resolved?

**Comment by franfernandez20** [comment]
Same issue detected 

![image](https://github.com/community/community/assets/23079185/81b7b154-fe5c-4e97-ab81-804762564613)
